src/TesterInternal/StreamingTests/SMSStreamingTests.cs

```diff
@@ -35,30 +35,23 @@ public class SMSStreamingTests : UnitTestSiloHost
                         StartSecondary = false, 
                     };
 
-        private readonly SingleStreamTestRunner runner;
-        private readonly bool fireAndForgetDeliveryProperty;
+        private SingleStreamTestRunner runner;
+        private bool fireAndForgetDeliveryProperty;
 
-        public SMSStreamingTests()
-            : base(smsSiloOption, smsClientOptions)
+        public static TestingSiloHost CreateSiloHost()
         {
-            runner = new SingleStreamTestRunner(SingleStreamTestRunner.SMS_STREAM_PROVIDER_NAME);
-            fireAndForgetDeliveryProperty = ExtractFireAndForgetDeliveryProperty();
+            return new TestingSiloHost(
+                smsSiloOption,
+                //smsSiloOption_OnlyPrimary,
+                smsClientOptions);
         }
 
-        public SMSStreamingTests(int dummy)
-            : base(smsSiloOption_OnlyPrimary, smsClientOptions)
```

Comment:
or maybe @gabikliot knows if we care about these non-default constructors... I think it's not worth spending any time on those, just highlighting that with this changes, the context of the old constructors might be lost, in case we cared about them. I suppose they were just some dev-only tests as the author was troubleshooting issues